starter-core/src/main/java/io/micronaut/starter/feature/json/SerializationFeature.java

```diff
@@ -15,12 +15,22 @@
  */
 package io.micronaut.starter.feature.json;
 
+import io.micronaut.core.annotation.NonNull;
 import io.micronaut.starter.application.ApplicationType;
 import io.micronaut.starter.application.generator.GeneratorContext;
 import io.micronaut.starter.build.dependencies.Dependency;
+import io.micronaut.starter.build.dependencies.Substitution;
 import io.micronaut.starter.feature.Category;
+import io.micronaut.starter.options.BuildTool;
+import java.util.ArrayList;
+import java.util.List;
 
 public interface SerializationFeature extends JsonFeature {
+    String MICRONAUT_SERIALIZATION = "micronaut.serialization";
+    String GROUP_ID_MICRONAUT = "io.micronaut";
+    String GROUP_ID_MICRONAUT_SERDE = "io.micronaut.serde";
+    String ARTIFACT_ID_MICRONAUT_JACKSON_DATABIND = "micronaut-jackson-databind";
```

Comment:
might as well make constants for the full Dependency object

Reply:
Something like:

```java
Dependency MICRONAUT_JACKSON_DATABIND = Dependency.builder()
                                .groupId(GROUP_ID_MICRONAUT)
                                .artifactId(ARTIFACT_ID_MICRONAUT_JACKSON_DATABIND)
                                .build();
```

Reply:
I extracted it since every JsonFeature uses it: 

https://github.com/micronaut-projects/micronaut-starter/pull/1189/commits/6826a62e77a6b48845b6f981e678f39aaa8dddea